static/js/main.js

```diff
@@ -37,9 +39,10 @@ function start(deployments, owner, repo, callback) {
   function bug_url(id) {
     return 'https://bugzilla.mozilla.org/show_bug.cgi?id=' + id;
   }
-  function bug_id(commit) {
-    var msg = commit.commit.message;
-    if (msg.match(/\d{6,7}/g)) return msg.match(/\d{6,7}/g)[0];
+  function bug_id(msg) {
+    if (msg.match(/\b\d{6,7}\b/g)) {
+      return msg.match(/\b\d{6,7}\b/g)[0];
+    }
```

Comment:
This is horrendously inefficient.

You should instead do:
```js
function bug_id(msg) {
	let result = msg.match(/\b\d{6,7}\b/g);
	return Boolean(result) && result[0];
}
```

Reply:
I don't know about "horrendously" :)
I agree that it's inefficient in theory. At least in a purely mathematical way. 

Also, a prettier solution would probably be:

```javascript
let result = msg.match(/\b\d{6,7}\b/g);
return result && result[0] || false;
```